doc/api/timers.md

```diff
@@ -263,6 +263,17 @@ added: v0.0.1
 
 Cancels a `Timeout` object created by [`setTimeout()`][].
 
+## Diagnostics
+
+### getActiveTimers()
+<!-- YAML
+added: REPLACEME
+-->
+
```

Comment:
This is missing the return type. Could you include it, even though the text says that an array is returned.